### PR TITLE
update links

### PR DESCRIPTION
**Pull request by Arhell**
resolve https://github.com/open-telemetry/opentelemetry.io/issues/1811

### CODE REVIEW

File: LICENSE-CODE [hunk 1]
```diff
@@ -1,7 +1,7 @@
 
                                  Apache License
                            Version 2.0, January 2004
-                        http://www.apache.org/licenses/
+                        https://www.apache.org/licenses/
```
Comment (chalin) [inline LICENSE-CODE:4]: The official license text uses http, and I'd like to match the official text, to the letter.
https://www.apache.org/licenses/LICENSE-2.0

I'd suggest undoing the changes to this file.

**Review by chalin — CHANGES_REQUESTED** [review]
Revert the changes to the license, otherwise LGTM

**Comment by svrnm** [comment]
What @chalin said rg. the license, beyond that lgtm, @Arhell: I assume you have verified that all the HTTPS urls work as well

File: LICENSE-CODE (hunk 1 above)
Comment (Arhell) [reply LICENSE-CODE]: Done

**Comment by Arhell** [comment]
new-end-user-resources.md add content after rebase

**Review by svrnm — APPROVED** [review]
lgtm.

@Arhell , thank you for your help once again!

**Pull request merged**